content/300-guides/200-deployment/110-deployment-guides/450-caveats-when-deploying-to-aws-platforms.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: 'Caveats when deploying to AWS platforms'
```

Comment:
What other platforms than AWS Lambda does this refer to?

Reply:
Ah, Beanstalk and RDS Proxy. You do not technically deploy to `RDS Proxy` so you might want to call this "using AWS services" instead.

Reply:
Or split "deploying to AWS platforms (Lambda, Beanstalk)" and "using a AWS database"